src/pages/PlaceList/PlaceList.jsx

```diff
@@ -0,0 +1,303 @@
+
+  useEffect(() => {
+    getPlaceList();
+  }, [houseTypte]);
+
+  const getPlaceList = async () => {
+    const pageNation = `${currentPage > 1 ? `offset=${currentPage}&limit=10` : `limit=10`}`;
+    const facilityFilter = toQueryString(facility, 'facility');
+    const houseFilter = toQueryString(houseTypte, 'type');
+
+    try {
+      setLodingSpinner(true);
+      const listData = await axios.get(
+        `http://192.168.235.22:8000/property?${pageNation}${facilityFilter && `&${facilityFilter}`}
+        ${houseFilter && `&${houseFilter}`}`
+      );
+      setPlaceList(listData.data.result);
+      setLodingSpinner(false);
+    } catch (err) {
+      console.log('err', err);
+      setLodingSpinner(false);
+    }
+  };
+
+  const toQueryString = (data, type) => {
+    if (type === undefined) {
+      const result = [];
+      Object.entries(data).forEach((filterData) => filterData[1] && result.push(filterData.join('=')));
+      return result && result.join('&');
+    } else {
+      const result = [];
+      data.forEach((filterData) => result.push(`${type}=${filterData}`));
+      return result && result.join('&');
+    }
+  };
+
+  return (
+    <PlaceListComponent>
+      <ListContainer>
+        <h4>300개 이상의 숙소</h4>
+        <h1>지도에서 선택한 지역의 숙소</h1>
+        <PlaceFilterPlaceList
+          houseTypte={houseTypte}
+          setHouseType={setHouseType}
+          modalState={modalState}
+          setModalState={setModalState}
+          facility={facility}
+          setfacility={setfacility}
+        />
+        <span className='notice'>여행 날짜와 게스트 인원수를 입력하면 1박당 총 요금을 확인할 수 있습니다.</span>
+        <div className='layoutLine'></div>
+        {placeList.length ? (
+          placeList.map((el, index) => {
+            return (
+              <PlaceItem key={index}>
+                <img className='placeImg' src={el.propertyImages} />
+
+                <div className='placeContent'>
+                  <div className='contentTitle'>
+                    <div>
+                      <div className='subTitle'>아직 모르는 데이터</div>
+                      <div className='title'>{el.propertyName}</div>
+                    </div>
+                    <div className='placeDibs'>{true ? <AiOutlineHeart /> : <AiFillHeart />}</div>
+                  </div>
+                  <div className='layoutLine'></div>
+                  <div className='option'>
+                    <span>최대인원 6명 ' 침실 2개 ' 침실 2개 ' 욕실 1개</span>
+                    <span>주방 ' 무료 주차 공간 ' 무선 인터넷 ' 난방</span>
+                  </div>
+                  <div className='placeValue'>
+                    <div className='placeGrade'>
+                      <AiFillStar className='plceGradeIcon' />
+                      <span className='plceGradeNumber'>4.64</span>
+                    </div>
+                    <div className='placePrice'>
+                      <span className='price'>{parseInt(el.price).toLocaleString()}</span>
```

Comment:
parseInt는 성능 이슈가 있습니다. Number()나 +를 사용해주세요.
```suggestion
                      <span className='price'>{Number(el.price).toLocaleString()}</span>
```

Reply:
그렇군요 좀더 한번 찾아 봐야겠내요